RobotCode/src/main/java/frc/robot/subsystems/ClimberSubsystem.java

```diff
@@ -18,6 +19,7 @@ public class ClimberSubsystem extends SubsystemBase
 	private ProfiledPIDController pidController;
 	private boolean override;
 	private double leftPower, rightPower;
+	private double gyroRoll;
```

Comment:
Doesn't need to be an instance variable. Just store it in the method when you're using it.